client/src/components/screens/PatientVariant/components/VariantNavigation.jsx

```diff
@@ -178,7 +178,7 @@ class VariantNavigation extends React.Component {
     let clickOutside = false;
     if (activeMenu.length > 0) {
       if (activeMenu[0].includes('overflowed-indicator') && !openMenu) openMenu = document.getElementById(`${activeMenu[0]}$Menu`);
-      openOverflowMenu = activeMenu.length > 1 ? document.getElementById(`${activeMenu[0]}$Menu`) : null;
+      openOverflowMenu = activeMenu.length > 1 ? document.getElementById('ant-menu-submenu') : null;
```

Comment:
Wouldn't it better to use `ref` here?

Reply:
After a couple of test  it's doesn't seems to be a quick change. I don't think it is worth it to do this change for now